build/AzurePipelinesTemplates/MUX-NugetReleaseTest-Job.yml

```diff
@@ -1,14 +1,25 @@
 parameters:
+  buildJobName: ''
+  runTestJobName: ''
   dependsOn: ''
-  useNupkgFromArtifacts: false
+  pkgArtifactPath: ''
   nugetConfigPath: test\MUXControlsReleaseTest\nuget.config
   matrix: 
-      Debug_x86:
-        buildPlatform: 'x86'
-        buildConfiguration: 'Debug'
+    Debug_x86:
+      buildPlatform: 'x86'
+      buildConfiguration: 'Debug'
+    Debug_x64:
+      buildPlatform: 'x64'
+      buildConfiguration: 'Debug'
+    Release_x64:
+      buildPlatform: 'x64'
+      buildConfiguration: 'Release'
+    Release_arm:
+      buildPlatform: 'arm'
+      buildConfiguration: 'Release'
```

Comment:
We should compile arm64 if we can (maybe wait until Ranjesh figures out his build issues first), but I want to add a backstop for arm64 as soon as we can.